packages/caver-rtm/src/index.d.ts

```diff
@@ -0,0 +1,269 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { ContractOptions, EventData } from '../../caver-contract/src'
+import { AccountKeyObject } from '../../caver-core-helpers/src/formatters'
+import AbstractTransaction from '../../caver-transaction/src/transactionTypes/abstractTransaction'
+import ValueTransfer from '../../caver-transaction/src/transactionTypes/valueTransfer/valueTransfer'
+import { SignatureObject } from '../../caver-utils/src/utils'
+
+export interface KlaytnAccount {
+    accType: number
+    account: {
+        balance: string
+        humanReadable: boolean
+        key: AccountKeyObject
+    }
+}
+
+export interface Block {
+    blockscore: string
+    extraData: string
+    gasUsed: string
+    governanceData: string
+    hash: string
+    logsBloom: string
+    number: string
+    parentHash: string
+    receiptsRoot: string
+    reward: string
+    size: string
+    stateRoot: string
+    timestamp: string
+    timestampFoS: string
+    totalBlockScore: string
+    transactions: TransactionFromNode[]
+    transactionsRoot: string
+    voteData: string
+}
+
+export interface BlockWithConsensusInfo extends Block {
+    committee: string[]
+    originProposer: string
+    proposer: string
+    round: number
+}
+
+export interface TransactionFromNode {
+    blockHash?: string
+    blockNumber?: string
+    codeFormat?: string
+    feePayer?: string
+    feePayerSignatures?: SignatureObject[]
+    feeRatio?: string
+    from?: string
+    gas?: string | number
+    gasPrice?: string | number
+    hash?: string
+    humanReadable?: boolean
+    key?: string
+    input?: string
+    nonce?: string
+    senderTxHash?: string
+    signatures?: SignatureObject[] | string[] | string[][]
+    to?: string | null
+    transactionIndex?: string | null
+    type?: string
+    typeInt?: number
+    value?: string
+    chainId?: string | number
+    data?: string
+}
```

Comment:
```suggestion
export interface TransactionFromNode {
    blockHash: string
    blockNumber: string
    codeFormat?: string
    feePayer?: string
    feePayerSignatures?: SignatureObject[]
    feeRatio?: string
    from: string
    gas: string | number
    gasPrice: string | number
    hash: string
    humanReadable?: boolean
    key?: string
    input?: string
    nonce: string
    senderTxHash: string
    signatures: SignatureObject[] | string[] | string[][]
    to: string | null
    transactionIndex: string | null
    type: string
    typeInt: number
    value?: string
}
```
I hope this can be used only for `caver.rpc.klay.getTransactionXXX`.
Then we can define most of fields as a required.

Reply:
Fixed. please check 04e9f11